acs/DnD.h

```diff
@@ -549,7 +554,7 @@ void DistributeBonus(int bonustype) {
 	int bval = 0, temp = 0, i = 0;
 	if(bonustype == BONUS_KILL) {
 		bval = CalculateBonus(BONUS_KILL);
-		temp = GetStat(STAT_LVLEXP) * bval / 100;
+		temp = LevelCurve[GetStat(STAT_LVL)] * bval / 100;
```

Comment:
This was intentionally set up this way, players get a % of the exp they acquired in that map (wording is confusing here I guess) not their own level.

Reply:
Without it, players won't get any exp. Nothing in the code uses STAT_LVLEXP.